modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -12894,6 +12894,7 @@ path-parameter-must-be-an-external-reference-code,-id,-or-unique-field=Path para
 path-parameter-property=Path Parameter Property
 path-parameters-are-not-supported-by-get-api-endpoints-with-collection-retrieve-type=Path parameters are not supported by GET API endpoints with "collection" retrieve type.
 path-parameters-are-not-supported-by-post-api-endpoints=Path parameters are not supported by POST API endpoints.
+path-parameters-cannot-be-established-without-a-response-schema-in-use=Path parameters cannot be established without a response schema in use.
```

Comment:
I prefer something easier in order to read: `Path parameter cannot be set without a response schema.`